examples/flags/readme.md

```diff
@@ -0,0 +1,105 @@
+
+  
+
+# Ignite App: Flags Example
+
+The `flags` Ignite App is a simple example demonstrating how to use command-line flags and arguments in Ignite applications.
+  
+
+## Installation
+
+  
+
+### Within Project Directory
+
+  
+
+To use the `flags` app within your project, execute the following command inside the project directory:
+
+  
+
+```bash
+
+ignite  app  install  github.com/ignite/apps/examples/flags
+
+```
+
+  
+
+The app will be available only when running `ignite` inside the project directory.
+
+  
+
+### Globally
+
+  
+
+To use the `flags` app globally, execute the following command:
+
+  
+
+```bash
+
+ignite  app  install  -g  github.com/ignite/apps/examples/flags
+
+```
+
+  
+
+This command will compile the app and make it immediately available to the `ignite` command lists.
+
+  
+
+## Requirements
+
+  
+
+- Go (version 1.16 or higher)
```

Comment:
I believe the example will work only for Go >= 1.21, a previous version might complain because the `go.mod` file format is different, specifically the `go 1.21.1` line:

```suggestion
- Go (version 1.21 or higher)
```